packages/dx-react-chart-material-ui/src/templates/tooltip/overlay.jsx

```diff
@@ -18,25 +19,46 @@ const styles = (theme) => {
   };
 };
 
-const popperModifiers = {
+const popperModifiers = arrowRef => ({
   flip: { enabled: false },
-};
+  arrow: {
+    element: arrowRef,
+  },
+});
 
-export const Overlay = withStyles(styles)(({
+const OverlayBase = ({
   classes, className, children, target, rotated, arrowComponent: ArrowComponent, ...restProps
 }) => {
+  const [arrowRef, setArrowRef] = React.useState(null);
```

Comment:
`useRef` is better for this task

Reply:
I meant React.useRef hook :)

Reply:
If we use `useRef`, then we should use `useEffect` and it is not looks better.